community/hsetroot/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+make DESTDIR="$(realpath "$1")" PREFIX=/usr install
```

Comment:
Why the `realpath`?

Reply:
libtool complains about relative paths

Reply:
`$1` isn't relative.

Reply:
oh, sorry, I was manually running build with a relative path.